.travis/prepare.sh

```diff
@@ -45,9 +45,8 @@ install_clang
 
 export PATH="/usr/local/clang/bin:$PATH"
 
-# disable go modules to avoid downloading all dependencies when doing go get
-GO111MODULE=off go get golang.org/x/tools/cmd/cover
-GO111MODULE=off go get github.com/mattn/goveralls
+go install golang.org/x/tools/cmd/cover@latest
```

Comment:
Accordingly the docs, this tools is now part of the std library, should we remove it?